docs/scipp-neutron/from-mantid-to-scipp.ipynb

```diff
@@ -465,7 +483,8 @@
     "    ['tof'],\n",
     "    values=np.arange(0.0, 20000.0, 100.0),\n",
     "    unit=sc.units.us)\n",
-    "histo = sc.rebin(histo, 'tof', edges)"
+    "data = sc.compat.mantid.from_mantid(input_bin_edges)\n",
```

Comment:
`sc.neutron.from_mantid`?

Reply:
i agree that using `neutron` is better than introducing a new `compat` submodule here